src/config.ts

```diff
@@ -3,22 +3,24 @@ import { ethers } from 'ethers';
 import { RouterConfig } from '@hyperlane-xyz/sdk';
 
 export type SyntheticConfig = {
-  type: "SYNTHETIC";
+  type: "SYNTHETIC" | "SYNTHETIC_URI";
```

Comment:
Make an enum out of this?

Reply:
And would suggest lower-case to better match JS convention, but nbd